meta/main.yml

```diff
@@ -10,7 +10,7 @@ galaxy_info:
     - centos
   company: Red Hat, Inc.
   license: GPL-3.0+
-  min_ansible_version: 2.5
+  min_ansible_version: 2.8
```

Comment:
Should we bump the min_ansible_version to 2.8 in other roles, as well? The oldest is 2.2... (firewall, postfix and tuned)

Reply:
> Should we bump the min_ansible_version to 2.8 in other roles, as well? The oldest is 2.2... (firewall, postfix and tuned)

I think we'll need to figure out what to do there.  It may be that they still work on old versions of ansible - and there may be some users who are stuck on some old version of ansible and use the role - but we cannot support it